README.rst

```diff
@@ -43,6 +43,30 @@ Single database server with empty database
                 host: 'localhost'
                 rights: 'all privileges'
 
+With custom settings
+
+.. code-block:: yaml
+
+    postgresql:
+      server:
+        ...
+        settings:
```

Comment:
we don't do it this way for a good reason. Rather structure the settings under postresql:server pillar directly. That will make pillars readable, structured etc. The formula user should not recognize whether it's value for config file, etc..

I personally would agree with kind of "generic" ad-hoc way to add additional settings that formula don't support, but as we don't have any described procedure for that, it will not pass other reviewers.